plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ProxyServicePropertiesEditionPartForm.java

```diff
@@ -1624,10 +1624,12 @@ public void selectionChanged(SelectionChangedEvent event) {
      */
 	protected Composite createWsdlXMLText(FormToolkit widgetFactory, Composite parent) {
 		Control wsdlXMLLabel = createDescription(parent, EsbViewsRepository.ProxyService.Wsdl.wsdlXML, EsbMessages.ProxyServicePropertiesEditionPart_WsdlXMLLabel);
-		wsdlXML = widgetFactory.createText(parent, ""); //$NON-NLS-1$
+//		wsdlXML = widgetFactory.createText(parent, "", SWT.V_SCROLL); //$NON-NLS-1$
```

Comment:
Remove the commented line.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/766